library/core/src/ffi.rs

```diff
@@ -223,7 +227,7 @@ impl<'f> VaListImpl<'f> {
 
 #[cfg(all(
     any(target_arch = "aarch64", target_arch = "powerpc", target_arch = "x86_64"),
-    any(not(target_arch = "aarch64"), not(target_os = "ios")),
+    any(not(target_arch = "aarch64"), all(not(target_os = "macos"), not(target_os = "ios"))),
```

Comment:
minor nit: this is another instance of `all(not(...), not(...))`

Reply:
Changed!